objectrocket/instances/__init__.py

```diff
@@ -14,7 +14,7 @@
 logger = logging.getLogger(__name__)
 
 
-class Instances(bases.BaseOperationsLayer):
+class Instances(bases.Extensible, bases.BaseOperationsLayer):
```

Comment:
We should keep the `BaseOperationsLayer` at the front of the bases declaration.

Per python's new-style class MRO, the left-most and lowest occurring class in a hierarchy will have its method selected first at resolution time. So, to ensure that the `BaseOperationsLayer.__init__` is always called first, we should keep it as the first item in this bases declaration. Under certain circumstances, the `__init__` could be completely skipped — say, if someone decided to add an `__init__` to the `Extensible` mixin, or added a new base to this class at the front of the bases declaration.